### ISSUE TITLE
CPSV-AP: Evidence sometimes generic, sometimes specific.

### ISSUE THREAD

**Issue by GeertThijs** [Informatievlaanderen/OSLO#25]
Problem with Evidence is that it is sometimes generic (like DriversLicense) and sometimes specific for the PublicService (eg PermitForFleaMarketInGhent). In the first case it is useful to model Evidence as a separate class, in the latter it would be better to model Evidence as an attribute of PublicService.  Similar problem for Output & CriterionRequirement.

**Comment by GeertThijs** [comment]
Eerlijk gezegd zou ik wel bij een class blijven voor Evidence, Output & CriterionRequirement. Een class met een externe Identifier voor elke instantie. Betekent niet alleen dat een bepaalde Output bv voor meerdere Public Services kan worden gebruikt in 1 en dezelfdse datauitwisseling, maar ook daarbuiten dankzij de ecxterne identifier. Alleen betekent dit dat elke gemeente waar een toelating nodig is voor bv een rommelmarkt een eigen ToelatingVoorRommelmarkt gaat hebben. Dat wordt dus een lange lijst met Output's (lees Producten).

**Comment by dhaeneth** [comment]
idd. Maar ik volg u volledig, Geert

**Comment by DieterDePaepe** [comment]
Eventueel kan je dit wat omzeilen met een `LocalPermitForFleaMarket`, of dat een goed idee is laat ik nog even in het midden.

**Comment by GeertThijs** [comment]
Nee, want ih voorbeeld verwijs ik naar het specifiek voor Gentse rommelmarkten vereiste aanvraagformulier, opgesteld door de stad Gent. In andere steden kan dat er compleet anders uitzien.
